### PR TITLE
feat: sentencizer

### PR DESCRIPTION
**Pull request by cristianmtr**
@cristianmtr 

AFTER MERGING

- add note in old
- archive old repo

### CODE REVIEW

File: jinahub/crafters/Sentencizer/manifest.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+manifest_version: 1
+name: sentencizer
+alias: Sentencizer
+description: The sentencizer splits text into sentences.
+author: Jina AI Dev-Team (dev-team@jina.ai)
+url: https://github.com/jina-ai/executor-text-sentencizer
```
Comment (jakob1111996) [inline jinahub/crafters/Sentencizer/manifest.yml:6]: wrong url

File: scripts/tests.sh [hunk 2]
```diff
@@ -21,7 +21,6 @@ if [[ -d "tests/" ]]; then
 
   if [[ -f "tests/requirements.txt" ]]; then
   pip install -r tests/requirements.txt
```
Comment (jakob1111996) [inline scripts/tests.sh:23]: don't we need a setup.py here?

**Comment by jakob1111996** [comment]
I am working on this.

File: scripts/tests.sh (hunk 2 above)
Comment (cristianmtr) [reply scripts/tests.sh:23]: We shouldn't be using `setup.py` anymore.

See discussions in Slack.

1. Executors are not meant to be pypi packages
2. do relative imports in tests

**Review by jakob1111996 — APPROVED** [review]
LGTM

**Pull request merged**